controllers/volumereplication_controller.go

```diff
@@ -87,6 +88,7 @@ func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	// Get VolumeReplicationClass
 	vrcObj, err := r.getVolumeReplicaCLass(instance.Spec.VolumeReplicationClass)
 	if err != nil {
+		_ = r.updateReplicationStatus(instance, replicationv1alpha1.ReplicationFailure, err.Error())
```

Comment:
It should be safe to remove `_` above if not using the return value here.

Reply:
As far as I know, linter fails in those cases. But let's try

Reply:
![Screenshot from 2021-03-17 15-15-10](https://user-images.githubusercontent.com/22993933/111447794-cbf75d00-8733-11eb-9773-ede0799d71d0.png)